pkg/handler/features.go

```diff
@@ -74,6 +74,18 @@ func CheckSnapshotAccessible(ctx context.Context) (err error) {
 		return nil
 	} else {
 		return ce.NewErrorResponse(http.StatusBadRequest, "Cannot manage repository snapshots",
-			"Neither the user nor the account is allowed.")
+			"Neither the user nor account is not allowed.")
```

Comment:
I think this was a typo from my original commit that i fixed but you may be reverting here